frontend/src/App.tsx

```diff
@@ -1,24 +0,0 @@
-import { Route, Router, Switch } from 'wouter';
```

Comment:
Why do we lowercase all filesnames?

Reply:
Nope, was just wondering because I always found it intuitive to have the main export as the filename